lib/raygun.ts

```diff
@@ -30,8 +30,23 @@ import { OfflineStorage } from "./raygun.offline";
 import { startTimer } from './timer';
 import * as raygunTransport from "./raygun.transport";
 
+import { v4 as uuidv4 } from "uuid";
+
 const debug = require("debug")("raygun");
 
+
+let apmBridge: undefined | null | {notify(e: string | Error, s: string): void} = undefined;
+
+try {
+  if (module.parent) {
+    apmBridge = module.parent.require('raygun-apm/lib/src/crash_reporting');
+  } else {
+    apmBridge = require('raygun-apm/lib/src/crash_reporting');
+  }
+} catch (e) {
+  apmBridge = null;
+}
+
```

Comment:
This looks a little complex, but in essence we need require the `raygun-apm` module from the parent's require scope to have access. If for any reason we aren't able to do that, linked exceptions will be disabled.